modules/tinymce/src/themes/silver/test/ts/browser/editor/header/SpapTest.ts

```diff
@@ -0,0 +1,35 @@
+import { UiFinder, Waiter } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { SugarElement } from '@ephox/sugar';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.header.SpapTest', () => {
+  const promotionClass = 'tox-spap';
+
+  context('spap turned off', () => {
+    const hook = TinyHooks.bddSetup<Editor>({
+      base_url: '/project/tinymce/js/tinymce',
+      promotion: false
+    }, []);
+
+    it('promotion should not be displayed', async () => {
```

Comment:
Tests should be prefixed with Jira numbers.

Reply:
Ah, completely forgot, that's my bad.